firebase-inappmessaging/src/main/java/com/google/firebase/inappmessaging/internal/InstallationIdResult.java

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.inappmessaging.internal;
+
+import android.text.TextUtils;
+import com.google.firebase.installations.InstallationTokenResult;
+
+public class InstallationIdResult {
```

Comment:
Ideally we should be using https://github.com/google/auto/blob/master/value/userguide/index.md for these value objects. If its not too much work, consider adding using this lib